Department.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace labs;
+
+class Department
+{
+    public array $employees;
+    public string $name;
+    function __construct($array_of_employee,$name){
+        $this->name = $name;
+        $this->employees = $array_of_employee;
+    }
+
+    public function SalarySum(): array
```

Comment:
название методов должно быть в camelCase
```suggestion
    public function salarySum(): array
```